perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -529,8 +537,10 @@ def Prepare(self, vm):
     vm.PushFile(FLAGS.boto_file_location, DEFAULT_BOTO_LOCATION)
 
     self.bucket_name = 'pkb%s' % FLAGS.run_uri
+
     vm.RemoteCommand(
-        'aws s3 mb s3://%s --region=us-east-1' % self.bucket_name)
+        'aws s3 mb s3://%s --region=%s' % (
```

Comment:
It's more complicated than this, you will also need to change the actual test script to use the correct endpoint for the region where the storage bucket is created in.

Here is a official table listing all the regions and endpoints:
http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region

Then in our code, we need to pass the region name to the actual test script which will then lookup the endpoint to use using the table above.

Similar logic is necessary for Azure, perhaps for Google (perhaps not).